web/frontend/src/pages/Home/AdvancedOptions.tsx

```diff
@@ -34,7 +34,7 @@ export default function AdvancedOptions(props: Props) {
 
   return (
     <Collapsible title="Advanced Options">
-      <div className={styles.FormGrid}>
+      <div className={styles.advanceoptionsformgrid}>
```

Comment:
The font looks different compared to figma

Reply:
just noticed the tailwind changes are to be addressed in different ticket
Current changes looks good